src/Linux/Avalonia.LinuxFramebuffer/Output/DrmBindings.cs

```diff
@@ -88,7 +88,8 @@ public DrmResources(int fd)
             if (res == null)
                 throw new Win32Exception("drmModeGetResources failed");
             
-            var crtcs = new drmModeCrtc[res->count_crtcs];
+            Span<drmModeCrtc> crtcs = stackalloc drmModeCrtc[res->count_crtcs];
```

Comment:
How many entries can we get from `count_crtcs` normally ?

Reply:
Stack allocation is always a risk when the length is not constant. In this case I think it won't be more than 10? In most cases I believe it will be between 1 and 3.

Reply:
`count_crtcs` is driver-specific, so I don't think it's a good candidate for stackalloc, also it might ba a wrong place for reducing allocations anyway since it only gets called once on application startup